src/Http/RedirectBinding.php

```diff
@@ -149,14 +150,118 @@ public function processSignedRequest(Request $request)
     }
 
     /**
+     * @param Request $request
+     * @return ReceivedAuthnRequest
+     */
+    public function receiveUnsignedAuthnRequestFrom(Request $request)
```

Comment:
As discussed, I will create a tech debt issue for this (and for the way AuthnRequests are built and used), so we can clean both concepts up a bit and release it in a new major.